### ISSUE TITLE
Non-Selected Subraces Become Features

### ISSUE THREAD

**Issue by b-rand** [HeroCreationLab/hero-creation-tool#157]
This line is meant to filter out subraces from features.
What actually happens is the current subrace is filtered out and any additional subraces are added as features.
https://github.com/HeroCreationLab/hero-creation-tool/blob/021ce7e5fabaea985635dc6f1a81be191a77a072/src/module/tabs/raceStep.ts#L60

I created a Mountain Dwarf subrace, when selecting this subrace, the Hill Dwarf subrace is added as a feature.
<img width="1013" alt="image" src="https://github.com/HeroCreationLab/hero-creation-tool/assets/19173630/708a2677-40b5-4f46-a710-f2ab8170b307">

**Comment by MythicalRollers** [comment]
You might have something not linked correctly.  My Hill, Mountain, and Mark of Warding Dwarf all have separate features.  I am not sure how you have yours set up, but mine is like this.  Base race Dwarf has all features linked to it which are shared between all 3 dwarves.  Then each subrace is linked to dwarf with the info specific to it listed.  Then any specific features are linked to them.  IE Mountain gets the training and Marked gets the spell list.

**Comment by ccjmk** [comment]
I am marking this bug as `wont-fix`  and closing it because Subraces are no longer going to be supported by any workaround, as the system itself decided to make them 1st-class objecfs as `Race`-type items and just incorporate parent class stuff in there.

TL;DR: `dnd5e` is getting rid of "Elf" and "High Elf" as two different objects and will now just have "High Elf" with stuff from both, so there will no be subclass-features to add or miss or accidentally add now!
